src/auto/constants.rs

```diff
@@ -17,12 +17,6 @@ pub static CSET_a_2_z: once_cell::sync::Lazy<&'static str> =
     once_cell::sync::Lazy::new(|| unsafe {
         CStr::from_ptr(glib_sys::G_CSET_a_2_z).to_str().unwrap()
     });
-pub static KEY_FILE_DESKTOP_ACTION_GROUP_PREFIX: once_cell::sync::Lazy<&'static str> =
```

Comment:
Strange, why it was removed?

Reply:
I wondered the same. Most of them seems to not be that useful or not used anymore

Reply:
It's a SuSE/Debian/Ubuntu-specific patch that added these in the first place :man_facepalming: 
https://sources.debian.org/src/glib2.0/2.62.4-2/debian/patches/01_gettext-desktopfiles.patch/